client/state/data-layer/wpcom/me/settings/index.js

```diff
@@ -86,6 +86,6 @@ export const finishUserSettingsSave = ( { dispatch }, { settingsOverride }, next
 };
 
 export default {
-	[ USER_SETTINGS_REQUEST ]: [ dispatchRequest( requestUserSettings, storeFetchedUserSettings, noop ) ],
-	[ USER_SETTINGS_SAVE ]: [ dispatchRequest( saveUserSettings, finishUserSettingsSave, noop ) ],
+	[ USER_SETTINGS_REQUEST ]: [ dispatchRequest( requestUserSettings, storeFetchedUserSettings ) ],
+	[ USER_SETTINGS_SAVE ]: [ dispatchRequest( saveUserSettings, finishUserSettingsSave ) ],
```

Comment:
case in point, we're simply ignoring errors in this existing code. sorry if I let that slide past review.